src/main/java/baseball/io/PlayerIoManager.java

```diff
@@ -0,0 +1,51 @@
+package baseball.io;
+
+import baseball.constant.NumberConstant;
+import baseball.io.console.ConsoleReader;
+import baseball.io.console.ConsoleWriter;
+import baseball.model.judge.JudgeResult;
+import baseball.utils.NumberValidator;
+import java.util.List;
+
+public class PlayerIoManager {
+
+    private ConsoleReader consoleReader;
+    private ConsoleWriter consoleWriter;
+
+    private PlayerIoManager(ConsoleReader consoleReader, ConsoleWriter consoleWriter) {
+        this.consoleReader = consoleReader;
+        this.consoleWriter = consoleWriter;
+    }
+
+    public static PlayerIoManager create() {
+        return new PlayerIoManager(new ConsoleReader(), new ConsoleWriter());
+    }
+
+    public void welcomePlayer() {
+        consoleWriter.printGameStartText();
+    }
+
+    public List<Integer> askThreeNumbers() {
+        consoleWriter.printNumberInputText();
+
+        List<Integer> numbers = consoleReader.readThreeNumbers();
+        NumberValidator.validateNumbersInRange(numbers, 1, 9);
+        NumberValidator.validateNotDuplicated(numbers);
+
+        return numbers;
+    }
+
+    public void notifyJudgeResult(JudgeResult judgeResult) {
+        consoleWriter.printLine(judgeResult.toResultString());
+    }
+
+    public void congratulate() {
+        consoleWriter.printCongratulationText();
+    }
+
+    public boolean isPlayerWantToReplay() {
+        consoleWriter.printReplayAskText();
+        return consoleReader.readOneOrTwo() == NumberConstant.ONE;
```

Comment:
재시도를 관리하는 enum으로 관리하는 건 어떨까요?
그러면 ONE, TWO 보다 더 명시적으로 입력받는 숫자 1, 2도 관리할 수 있을 것 같아요!

```java
public enum Retry {
    RETRY("1"), QUIT("2");
}
```

```suggestion
        return consoleReader.readOneOrTwo() == Retry.RETRY;
```